utils/verifySecretKey.ts

```diff
@@ -1,7 +1,7 @@
 import config from '../config'
 import lang from './lang'
 
-const verifySecretKey = async (secret: string): Promise<void> => {
+const verifySecretKey = (secret: string): void => {
   if (secret !== config.get('secret.key')) throw Error(lang.__('credential_failed'))
 }
```

Comment:
just for discuss, I think checker function like this better if we considering to return an error even just using only bool, or (error, bool). wondering if this function can be more reuseable for any usecase.

so maybe u can call this function like this
```
if isVerfiedSecretKey {
 ....
}
```